app/helpers/claims_helper.rb

```diff
@@ -1,12 +1,12 @@
 module ClaimsHelper
-  def verified_fields(claim)
+  def govuk_verify_fields(claim)
```

Comment:
Maybe a better name for this would be `fieldnames_from_govuk_verify` to differentiate it from `claim.verified_fields`